docs/source/publishing-releases.rst

```diff
@@ -136,7 +136,8 @@ wait to publish a release until your package is usable and tested.
 
       git clean -dfx
 
-#. Publish a release on PyPI.
+#. Publish a release on PyPI. Note that you might need to configure
+   your `~/.pypirc` with a login token.
```

Comment:
Sphinx needs 2 back-ticks, unlike markdown. Also, it would be nice to provide a relevant documentation link.
```suggestion
   your ``~/.pypirc`` with a login token. See `the packaging documentation <https://packaging.python.org/specifications/pypirc/>_` for more details.
```